.dictionary

```diff
@@ -51,7 +54,9 @@ UTF
 UUID
 UUIDs
 Xcode
+XCode
```

Comment:
Apple [seems to spell it "Xcode"](https://developer.apple.com/xcode/)

Reply:
Also, I don't see this word mentioned in this PR, so not sure why it was required...

Reply:
True, it's in the changelog. Not sure why it wasn't caught, I guess fixing it there would be the better option.

Reply:
No worries.  "/me puts away his pedantic hat..."